test/runtime/ct.go

```diff
@@ -0,0 +1,433 @@
+package RuntimeTest
+
+import (
+	"fmt"
+	"net"
+	"strings"
+
+	"github.com/asaskevich/govalidator"
+	"github.com/cilium/cilium/test/helpers"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/types"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	serverImage = "httpd"
+	ctCleanUpNC = "ct-clean-up-nc.py"
+	ctL4Policy  = "CT-l4-policy.json"
+	netcat      = "netcat"
+)
+
+type connTest struct {
+	src         map[string]string
+	destination map[string]string
+	kind        string
+	mode        string
+}
+
+func (c connTest) String() string {
+	return fmt.Sprintf("%s-%s-%s", c.src["Name"], c.destination["Name"], c.kind)
+}
+
+var _ = Describe("RuntimeConntrackTable", func() {
+
+	var initialized bool
+	var logger *logrus.Entry
+	var vm *helpers.SSHMeta
+	var (
+		HTTPPrivate   = "private"
+		HTTPPublic    = "public"
+		HTTPDummy     = "dummy"
+		netcatPrivate = "ncPrivate"
+		netcatPublic  = "ncPublic"
+		server        = "server"
+		server2       = "server-2"
+		server3       = "server-3"
+		client2       = "client-2"
+		client        = "client"
+	)
+
+	initialize := func() {
+		if initialized == true {
+			return
+		}
+		logger = log.WithFields(logrus.Fields{"testName": "RuntimeConntrack"})
+		logger.Info("Starting")
+		vm = helpers.CreateNewRuntimeHelper(helpers.Runtime, logger)
+		vm.WaitUntilReady(100)
+		vm.NetworkCreate(helpers.CiliumDockerNetwork, "")
```

Comment:
Why is the resultant `*CmdRes` ignored here?

Reply:
The thing is that NetworkCreate sometimes failed that it's already created. Maybe added a function like now (NetworkCreateIfNotExists)?

Reply:
+1, that function would be helpful to have.